db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -2156,7 +2164,14 @@ Status DBImpl::FlushMemTable(ColumnFamilyData* cfd,
 Status DBImpl::AtomicFlushMemTables(
     const autovector<ColumnFamilyData*>& column_family_datas,
     const FlushOptions& flush_options, FlushReason flush_reason,
-    bool writes_stopped) {
+    bool entered_write_thread) {
+  assert(immutable_db_options_.atomic_flush);
+  if (write_controller_.IsStopped() && !flush_options.wait) {
```

Comment:
nit: I would probably revert the two conditions. Although right now write_controller_.IsStopped() is a simple and cheap call, it could become more complicated.

Reply:
Done